app/aws/metrics.py

```diff
@@ -67,20 +69,22 @@ def put_batch_saving_inflight_processed(metrics_logger: MetricsLogger, count: in
         metrics_logger.set_namespace("NotificationCanadaCa")
         metrics_logger.put_metric("batch_saving_inflight", count, "Count")
         metrics_logger.set_dimensions({"acknowledged": "True"})
+        metrics_logger.set_dimensions({"notification_type": queue._suffix or "none"})
+        metrics_logger.set_dimensions({"priority": queue._process_type or "none"})
```

Comment:
@sastels can all the `or "none"` code be changed to the below ? It'll reduce costs and better follows how metrics dimensions should be created.

```python
if queue._suffix:
   metrics_logger.set_dimensions({"notification_type": queue._suffix})
```